devtools/rabbitloop/claude.py

```
@@ -0,0 +1,135 @@
+            comment.file_path,
+            comment.line,
+        )
+        logging.debug("Prompt:\n%s", prompt)
+        return FixResult(completed=False)
+
+    logging.info("Invoking claude for %s:%s", comment.file_path, comment.line)
+
+    head_before = _get_head_sha()
+
+    try:
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)